archetypal/idfclass.py

```diff
@@ -123,6 +123,30 @@ def partition_ratio(self):
 
         return partition_lineal / self.area_conditioned
 
+    @property
```

Comment:
This is a good way, although I think it would be better to return the wwr by orientation (azimuth).